tests/functional/cylc-cat-log/09-cat-running/flow.cylc

```diff
@@ -9,15 +9,29 @@
 
 [scheduling]
     [[graph]]
-        R1 = local-task:echo & remote-task:echo => cat-log
+        R1 = """
+            # start "local-task" and "remote-task", then run cat-log
+            local-task:echo & remote-task:echo => cat-log
+
+            # add a downstream task to ensure "local-task" and "remote-task"
+            # succeed (for the reference test)
+            local-task & remote-task => fin
+        """
 
 [runtime]
     [[ECHO]]
         script = """
             cylc__job__wait_cylc_message_started
             echo rubbish
             echo garbage >&2
-            cylc message 'echo done'
+            cylc message -- 'echo done'
+
+            # wait up to PT1M for the cat-log task to succeed
+            # (the workflow will shut down if cat-log fails)
+            sleep 60
+
+            # fail if the task was not orphaned by this point
+            false
```

Comment:
This test was supposed to be testing `cat-log` against running tasks:

> Test "cylc cat-log" of currently-running local and remote jobs.

However, there was nothing to keep the task running, so it was actually testing succeeded jobs.

I've changed the approach, the tasks now sleep for 60s, then fail.

If the `cat-log` works, it will `cylc set` the task, orphaning the sleep and triggering `fin` which is used to diagnose test success.

Reply:
Matt did purposefully try to pull out sleeps from the tests in order to speed them up reduce flakyness. It might have vanished then.

In this particular case, the sleep is ok.